.github/CONTRIBUTING.md

```diff
@@ -46,7 +46,7 @@ There is no need to squash your work as we will do that when the maintainer merg
   ```
   type(scope):desc
   ```
-  **Type** can be `fix`, `feat`, `docs`, `test` or `refactor`.
+  **Type** can be `chore`, `feat`, `docs`, or `test`.
```

Comment:
Do we need to add breakingChange or such? @jeroenptrs

Reply:
In my opinion test should be included in all upcoming PR's after we've decided on the muon thing, so actual PR's with "test" would be very few and thus we should put it in chore indeed 😄

Reply:
feat can be both breaking or non breaking - they can just add the X of breaking change into the PR template

Reply:
Fine by me 👍